monkestation/code/game/machinery/cloning.dm

```diff
@@ -55,6 +55,8 @@
 	var/auto_clone = TRUE
 	fair_market_price = 5 // He nodded, because he knew I was right. Then he swiped his credit card to pay me for arresting him.
 	payment_department = ACCOUNT_MED
+	/// Used in experimental cloner
```

Comment:
move this var down to just be on the exp cloner, as the only place its used on parent here is examine, which can also be moved down

Reply:
I moved it down, not sure if the way I did it was the best way though.

Reply:
Actually I could use it for reducing the speed for an evil cloner, so I might move it back up.